include/functions_csr.php

```diff
@@ -476,7 +476,7 @@ function sign_csr($passPhrase,$my_csrfile,$my_days,$my_device_type) {
 $config=$_SESSION['config'];
 $name = base64_encode(substr($my_csrfile, 0,strrpos($my_csrfile,'.')));
 $ext = substr($my_csrfile, strrpos($my_csrfile,'.'));
-$my_base64_csrfile=$name.$ext;
+$my_base64_csrfile = $my_csrfile //$name.$ext;

```

Comment:
@cuediin is there a reason that you split $name and $ext and then combined them without the "." between them? If not, this format might make more sense.